egs/wsj/s5/utils/prepare_lang.sh

```diff
@@ -275,6 +290,14 @@ echo $ndisambig > $tmpdir/lex_ndisambig
 
 ( for n in `seq 0 $ndisambig`; do echo '#'$n; done ) >$dir/phones/disambig.txt
 
+# In case there are extra word-level disambiguation symbols they also
+# need to be added to the list of phone-level disambiguation symbols.
+if [[ ! -z $extra_word_disambig_syms ]]; then
```

Comment:
just for stylistic consistency, please use single brackets `[ ]` unless double brackets are required by some special construct.